poucave/config.py

```diff
@@ -33,6 +33,7 @@
 REQUESTS_MAX_RETRIES = config("REQUESTS_MAX_RETRIES", default=2, cast=int)
 REQUESTS_MAX_PARALLEL = config("REQUESTS_MAX_PARALLEL", default=16, cast=int)
 SENTRY_DSN = config("SENTRY_DSN", default="")
+SOURCE_URL = config("SOURCE_URL", default="https://github.com/mozilla-services/poucave")
```

Comment:
I'm wondering if this should be a per-deployment URL or a per-task URL. I think I'm OK with per-deployment. If we need per-task in the future, it could default to the per-deployment `SOURCE_URL` without breaking existing code.